### ISSUE TITLE
Need advice in using utp-native for Android

### ISSUE THREAD

**Issue by paramtatv** [janeasystems/nodejs-mobile#158]
Hi,

utp-native works well in iOS but doesnt compile in Android
Is there a way to compile utp-native in Android?
Error is following:
`
```
platforms/android/build/nodejs-native-assets-temp-build/nodejs-native-assets-armeabi-v7a/nodejs-project/node_modules/utp-native
npm info lifecycle utp-native@2.1.3~install: Failed to exec install script
npm verb stack Error: utp-native@2.1.3 install: `node-gyp-build`
```

`

**Comment by paramtatv** [comment]
I found this solution but not sure for how to edit the common.gypi
[COMMON.GYPI Fix](
https://stackoverflow.com/questions/52878047/unable-to-link-libraries-when-compiling-native-node-modules-to-android-recompil/54032442#54032442)

Any help plz

**Comment by ralphtheninja** [comment]
Might want to make this post on `utp-native` repo.

**Comment by jaimecbernardo** [comment]
Hi @tim-coin,

The fix for the specific stack overflow error should've been fixed in `nodejs-mobile-gyp` v0.2.1 release: https://github.com/janeasystems/nodejs-mobile-gyp/commit/fa19d39733f08cf121479de8a0a6985ebc815e8a

Check which nodejs-mobile-gyp version is installed as a dependency of the local `nodejs-mobile-cordova` plugin. (Either in `node_modules/nodejs-mobile-gyp/` or `node_modules/nodejs-mobile-cordova/node_modules/nodejs-mobile-gyp/` )

**Comment by paramtatv** [comment]
Thanks,
